pkg/etcdenvvar/etcd_env.go

```diff
@@ -272,25 +271,9 @@ func getCipherSuites(envVarContext envVarContext) (map[string]string, error) {
 // getMaxLearners reads the control-plane replicas from the install-config. We tolerate max learners equal to the
 // desired control-plane replicas so that the admin can surge up N x 2 in case of vertical scaling/replacement.
 func getMaxLearners(envVarContext envVarContext) (map[string]string, error) {
-	clusterConfig, err := envVarContext.configmapLister.ConfigMaps(operatorclient.TargetNamespace).Get(clusterConfigName)
+	replicaCount, err := ceohelpers.GetMastersReplicaCount(envVarContext.configmapLister)
```

Comment:
remove `clusterConfigName, clusterConfigKey and replicaCountDecoder` from this file.